app/styles/components/gleam-logo.css

```diff
@@ -0,0 +1,12 @@
+.gleam-logo-container {
```

Comment:
@Arpan-206 all of this can be done via tailwind, you don't need specific styles for this

Reply:
right!